01-Primitives/05-Box-geometry.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<p>Question: Why the <code>fresnel.geometry.Box(scene, my_box)</code> in the middle?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/fresnel-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Do you mean why is the box in the middle or why do we not do something like  <code>thick_box = fresnel.geometry.Box(scene, my_box)</code></p><p> </p>

Reply:
Let me rephrase: Why show two boxes in one scene?

Reply:
Got it!